scripts/globals/abilities/pets/mountain_buster.lua

```diff
@@ -24,4 +24,11 @@ function onPetAbility(target, pet, skill)
 	target:updateEnmityFromDamage(pet,totaldamage);
 
 	return totaldamage;

```

Comment:
Nope, you still missed something important while reading the other script(s): returns end the scripts execution, it has to be last, move it right before the last "end".
Also, don't use tabs. Format with 4 spaces and () your conditions for readability.
```lua
    if (damage > 0) then
        if (target:hasStatusEffect(EFFECT_BIND) == false) then
            target:addStatusEffect(EFFECT_BIND, 1, 0, duration);
        end
    end
    return totaldamage;
end
```